Extends.md

```diff
@@ -0,0 +1,20 @@
+# Struct Inheritance for WGSL
+
+(TBD)
+
+It’s convenient to be able to express structures as combinations of other structures as in Java/TypeScript.
+
+wgsl-linker currently supports an `extends` syntax for struct inheritance (but `extends` is only lightly
```

Comment:
Interesting. Extends kind of has the same semantics here but on a struct level as what I was proposing for modules

Reply:
looking forward to understanding the modules idea, more to read.